Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+//Floyd-Warshall Algorithm is an algorithm for finding the shortest path between all the pairs of vertices in a weighted graph.
+
+$f    = fopen('php://stdin', 'r');
```

Comment:
please remove the extra space before equal sign

Reply:
@asha15 Done the required changes. Please review.